### ISSUE TITLE
MATSim 12.0 is not working on Windows 10

### ISSUE THREAD

**Issue by radhareddy31** [matsim-org/matsim-code-examples#451]
Hello, I am trying to run the MATSim jar file (on Windows 10) as mentioned in the book. The jar file is opening but not showing any tabs "choose, open, delete" to perform simulations.

**Comment by Janekdererste** [comment]
Hi, since I almost never run Matsim this way, could you summarize where you are downloading the jar file from and how you try to run it?

Maybe this is a way to go for you as well? https://github.com/matsim-org/matsim-example-project#building-and-running-it-locally

**Comment by radhareddy31** [comment]
Hello, Many thanks for your kind response. 
I have downloaded the MATSim zip file from https://matsim.org/downloads/.
I started running MATSim as specified in Chapter2, page 9 http://ci.matsim.org:8080/job/MATSim-Book/ws/partOne-latest.pdf.
When I double click on the matsim.jar file, it is opening without choose, open or delete tabs, to open config files to run simulations.

**Comment by JWJoubert** [comment]
One alternative thing to check is to ensure that your Jar-file launcher (the application that decides what to do when you double-click a jar-file) uses Java 11. This was an issue on Mac machines (and maybe something similar is happening on your Windows machine).

**Comment by radhareddy31** [comment]
Hello Joubert, 
Yes, indeed, it is showing that - You are using Java version:11.0.5; Oracle Corporation; mixed mode; 64-bit.

**Comment by mrieser** [comment]
"You are using Java version:" is part of the GUI, so the GUI is apparently showing. Maybe, for whatever reasons, the window is too small? Did you try to resize the window?
If that does not help, can you please post a screenshot so we can get a better understanding of how it looks like on your machine?

**Comment by radhareddy31** [comment]
Hello, I resize it from small to full screen, but no use. 
Please check the attachment for the display screen.
<img width="572" alt="MATSim" src="https://user-images.githubusercontent.com/58886102/97051086-764cc900-1576-11eb-9d26-1882425b5ae3.png">

**Comment by Janekdererste** [comment]
Indeed the 'choose' button is on the right of the text field for configuration file. Usually one has to make the window bigger since it seems to work with fixed widths. 

Alternatively you can:
1. Hold Shift and right click on your configuration file
2. Select copy as path
3. Paste the path into the text field next to Configuration file. 

Then the 'Start Matsim' Button should become enabled

**Comment by radhareddy31** [comment]
Many thanks, but changing the window size has no effect on the jar file.

I can not try the second option, as the Configuration File box is not responding / not taking the cursor.

**Comment by francescociari** [comment]
Hi, I have a similar problem, but in my case the GUI doesn't even open up. I went through the basics steps like ensuring that this is the right Java version and the fact that Java 11 is the default to run jar files. Any idea what could that be?
Thanks, 
Francesco

**Comment by radhareddy31** [comment]
Hello Francesco, I too have stuck with the issue, the jar file opens the window, but it behaves like a .jpg figure by not taking any input. I really do not understand, why this.

**Comment by mrieser** [comment]
Looking at the screenshot above, I realized that the font in the GUI is much bigger compared to the font of the window title. Are you using interface scaling in Windows? (E.g. the option to have "Larger Text" somewhere in Windows' display settings)

I would have assumed it should work, especially in newer Java versions like Java 11, but maybe this is the culprit.

**Comment by radhareddy31** [comment]
The window was opened in the Laptop and display is connected to the monitor, thats maybe the reason for differences in visibility. But, no settings were changed that might affect Java or MATSim.

**Comment by francescociari** [comment]
I finally solved the problem using this: https://johann.loefflmann.net/en/software/jarfix/index.html
Meaning that I don't know why it was wrong, but at least it worked in the end. 
I previously found this https://stackoverflow.com/questions/10446986/double-clicking-jar-file-does-not-open-command-prompt
that seemed to me to provide a plausible explanation of the problem (considering that I messed up a bit with Java versions before installing Java 11 and desinstalling newer versions, which might have caused wrong assignments to open a file  of a certain extension) but nothing worked out. But I hope the one or the other link here can help people encoutering the same problem in the future...
